### PR TITLE
Fix config

### PR DESCRIPTION
**Pull request by whilefoo**
@pavlovcik I implemented the custom array merger so in case of strings it concats arrays and removes duplicates and in case of objects just concats.
I changed `commands` to `disabledCommands` as array of strings and I changed time and priority labels to array of strings

### CODE REVIEW

File: src/utils/generate-configuration.ts [hunk 1]
```diff
@@ -1,75 +1,44 @@
...
+  const orgConfig = parseYaml(
     await download({
       context,
       repository: UBIQUIBOT_CONFIG_REPOSITORY,
       owner: payload.organization?.login || payload.repository.owner.login,
     })
   );
 
-  const repositoryConfiguration = parseYaml(
+  const repoConfig = parseYaml(
```
Comment (pavlovcik) [inline src/utils/generate-configuration.ts:23]: I prefer that you do not shorten words. The reason why is to reduce cognitive overhead when reading through all of the code.

**Review by 0x4007 — APPROVED** [review]
Thank you for your adjustments @whilefoo

**Comment by pavlovcik** [comment]
@wannacfuture please sync your branch.

File: src/types/configuration-types.ts [hunk 2]
```diff
@@ -101,13 +95,7 @@ export const BotConfigSchema = StrictObject(
...
       issueCreatorMultiplier: T.Number({ default: 1 }),
     }),
-    commands: T.Array(
-      StrictObject({
-        name: T.String(),
-        enabled: T.Boolean(),
-      }),
-      { default: allCommands }
-    ),
+    disabledCommands: T.Array(T.String(), { default: allCommands }),
```
Comment (0x4007) [inline src/types/configuration-types.ts:98]: Perhaps it makes sense to enable all commands by default? It seems that disabling by default has always caused more headaches when starting new repos for us? rfc @whilefoo

**Pull request merged**